Update node-providers.md (#976)
matter-labs/zksync-web-era-docs: docs/build/tooling/node-providers.md

### Ankr

> [Ankr](https://www.ankr.com/rpc/zksync/) provides private and public RPC endpoints for zkSync, powered by a globally distributed and decentralized network of nodes. They offer free and [paid plans](https://www.ankr.com/rpc/pricing/) with increased request limits.

Reviewer: ```suggestion
[Ankr](https://www.ankr.com/rpc/zksync_era/) provides private and public RPC endpoints for zkSync, powered by a globally distributed and decentralized network of nodes. They offer free and [paid plans](https://www.ankr.com/rpc/pricing/) with increased request limits.
```

### BlockPI

> [BlockPI](https://blockpi.io/) is a high-quality, robust, and efficient RPC service network that provides access to zkSync nodes with [free and paid plans](https://docs.blockpi.io/documentations/pricing).

Reviewer: ```suggestion
[BlockPI](https://blockpi.io/zksync) is a high-quality, robust, and efficient RPC service network that provides access to zkSync nodes with [free and paid plans](https://docs.blockpi.io/documentations/pricing).
```

### Chainstack

[Chainstack](https://chainstack.com/) is a leading blockchain infrastructure providing company. Build, run and scale blockchain applications using Chainstack nodes.

> ### ChainzkSync

Reviewer: ```suggestion
### Chainbase
```

> [ChainzkSync](https://chainzkSync.com/) API is supercharged for reliability, data correctness, and scalability. ChainzkSync will handle all the forks, upgrades, and network interruptions

Reviewer: ```suggestion
[Chainbase](https://chainbase.com/chainNetwork/zkSync) API is supercharged for reliability, data correctness, and scalability. Chainbase will handle all the forks, upgrades, and network interruptions
```

### DRPC

[DRPC](https://drpc.org/public-endpoints/zksync) offers access to distributed network of independent third-party partners and public nodes for zkSync. They provide a free tier that allows for an unlimited amount of requests over public nodes, or a paid tier which provides access to all providers, as well as other additional features.

### Quicknode

[QuickNode](https://www.quicknode.com/chains/zkSync) offers access to hosted zkSync nodes as part of their free Discover Plan. You can configure add-ons, like "Trace Mode" and "Archive Mode" for an additional cost by upgrading to one of their paid plans.

### Unifra

> [Unifra](https://zkSync.unifra.io/) is a Web3 developer platform that provides tools, APIs, and node infrastructure, and provides access to zkSync nodes that are nodes are reliabile, scalable, and easy to use.

Reviewer: ```suggestion
[Unifra](https://unifra.io/) is a Web3 developer platform that provides tools, APIs, and node infrastructure, and provides access to zkSync nodes that are nodes are reliabile, scalable, and easy to use.
```
